Meta/Lagom/Tools/CodeGenerators/LibGL/GenerateGLAPIWrapper.cpp

```diff
@@ -0,0 +1,570 @@
+/*
+ * Copyright (c) 2022, Jelle Raaijmakers <jelle@gmta.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Array.h>
+#include <AK/DeprecatedString.h>
+#include <AK/JsonObject.h>
+#include <AK/NumericLimits.h>
+#include <AK/Optional.h>
+#include <AK/SourceGenerator.h>
+#include <AK/StringBuilder.h>
+#include <AK/StringView.h>
+#include <AK/Vector.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/Stream.h>
+#include <LibMain/Main.h>
+
+struct ArgumentDefinition {
+    Optional<DeprecatedString> name;
+    Optional<DeprecatedString> cpp_type;
+    DeprecatedString expression;
+    Optional<DeprecatedString> cast_to;
+};
+
+struct FunctionDefinition {
+    DeprecatedString name;
+    DeprecatedString return_type;
+    Vector<ArgumentDefinition> arguments;
+    DeprecatedString implementation;
+    bool unimplemented;
+    DeprecatedString variant_gl_type;
+};
+
+struct VariantType {
+    DeprecatedString encoded_type;
+    Optional<DeprecatedString> implementation;
+    bool unimplemented;
+};
+
+struct Variants {
+    Vector<DeprecatedString> api_suffices { "" };
```

Comment:
Should this be spelled suffixes? (And the other few places this term shows up)
```suggestion
    Vector<DeprecatedString> api_suffixes { "" };
```

Reply:
Done!